nix/default.nix

```diff
@@ -1,6 +1,10 @@
-{ pkgs ? import <nixpkgs> {} }:
-
 let
+  nixpkgsRev = "49996e17ae9daf3c3447fb5e8500dc78b4fde995";
```

Comment:
actually we should still have `pkgs` as an argument and also read sha and rev from json file. we should have a please command to update this json files.